doc/whatsnew/2/2.15/index.rst

```diff
@@ -27,6 +27,10 @@ Extensions
 False positives fixed
 =====================
 
+* Avoid reporting ``unnecessary-dict-index-lookup`` or ``unnecessary-list-index-lookup``
+  when the index lookup is part of a destructuring assignment.
+
+  Closes #6788
```

Comment:
As a false positive fix it's eligible for a 2.14 backport. Do you mind moving this to 2.14.rst?

Reply:
I've moved it, but I guess the code will also need to be rebased. I don't mind doing that, but I don't want to rewrite history in this and mess up the code review.